docs/event-mesh/10-01-troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Troubleshooting
+---
+
+## Events do not reach the Namespace with active Eventing
+
+The upgrade of the Event Sources Controller Manager triggers the new Pod to update the KService for the HTTPSource. As a result of changes introduced to the KService, the Knative Serving Controller (KSC) creates a new Revision. The KSC does not pick up this new Revision and marks the KService as `Not ready`, still looking for the old Revision.
```

Comment:
The upgrade of the Event Sources Controller Manager triggers the update of the KService for the HTTPSource. As a result of changes introduced to the KService, the Knative Serving Controller (KSC) creates a new Revision. The KSC does not pick up this new Revision and marks the KService as `Not ready`, while looking for the old Revision.